bench/CMakeLists.txt

```diff
@@ -11,63 +11,58 @@ include(FindOpenCL)
 include(FindComputeCpp)
 include(ConfigureSYCLBLAS)
 
-# Manually search for CLBLAST
-# if (DEFINED CLBLAST_ROOT) 
-#   set(CMAKE_PREFIX_PATH ${CLBLAST_ROOT})
-# else() 
-#   set(CMAKE_PREFIX_PATH "/usr/lib/clblast")
-# endif()
-
-# pkg_search_module(CLBLAST clblast)
-# pkg_check_modules(CLBLAST clblast)
-
-# if (CLBLAST_FOUND)
-#   add_executable(clblast_benchmarks clblast_benchmark.cpp)
-#   set_property(TARGET clblast_benchmarks PROPERTY CXX_STANDARD 11)
-#   include_directories(${CLBLAST_INCLUDE_DIRS})
-#   target_link_libraries(clblast_benchmarks PUBLIC ${CLBLAST_LIBRARIES})
-#   target_link_libraries(clblast_benchmarks PUBLIC ${OpenCL_LIBRARIES})
-# endif()
-
-# # Manually search for CLBLAS_ROOT
-# if (DEFINED CLBLAS_ROOT) 
-#   set(CMAKE_PREFIX_PATH ${CLBLAS_ROOT})
-# else ()
-#   set(CMAKE_PREFIX_PATH "/usr/lib/clblas")
-# endif()
-
-# pkg_search_module(CLBLAS clBLAS)
-# pkg_check_modules(CLBLAS clBLAS)
+# Add syclblas benchmarks.
+file(GLOB SYCL_BENCHMARK_SRCS
+  ${CMAKE_CURRENT_SOURCE_DIR}/syclblas/syclblas_benchmark_level1.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/syclblas/syclblas_benchmark_level2.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/syclblas/syclblas_benchmark_level3.cpp
+)
+foreach(syclblas_benchmark ${SYCL_BENCHMARK_SRCS})
+  get_filename_component(sycl_bench_exec ${syclblas_benchmark} NAME_WE)
+  set(sycl_bench_exec, ${syclblas_benchmark})
```

Comment:
why there is a comma there?

Reply:
It appears to be shorthand for "TARGET", i.e. the following are equivalent: 
`set(sycl_bench_exec, ${syclblas_benchmark})`
`set(TARGET sycl_bench_exec ${syclblas_benchmark})`
I'll leave it as the latter, as it should make the intent clearer to later readers.